osu.Game/Online/WebSockets/WebSocketServer.cs

```diff
@@ -112,7 +121,7 @@ private async Task handleRequest(CancellationToken token)
                 // An HttpListenerException with the error code 995 (ERROR_OPERATION_ABORTED) is thrown
                 // when we call HttpListener.Stop on another thread while HttpListener.GetContextAsync is
                 // blocking in this thread. It is safe to ignore.
-                if (!(ex is HttpListenerException hx && hx.ErrorCode == 995))
+                if (!(ex is HttpListenerException hx && hx.ErrorCode == 995 || ex is ObjectDisposedException))
```

Comment:
Adding parenthesis in the first conditional check should satisfy code inspection.
```c#
if (!((ex is HttpListenerException hx && hx.ErrorCode == 995) || ex is ObjectDisposedException))
{
    throw;
}
```